tests/lib/rules/jsx-wrap-multilines.js

```diff
@@ -1393,16 +1393,6 @@ ruleTester.run('jsx-wrap-multilines', rule, {
         { messageId: 'parensOnNewLines' },
       ],
     },
-    {
-      code: DECLARATION_TERNARY_PAREN_FRAGMENT,
-      features: ['fragment'],
-      output: addNewLineSymbols(DECLARATION_TERNARY_PAREN_FRAGMENT),
-      options: [{ declaration: 'parens-new-line' }],
-      errors: [
-        { messageId: 'parensOnNewLines' },
-        { messageId: 'parensOnNewLines' },
-      ],
-    },
```

Comment:
Duplicate of:

https://github.com/jsx-eslint/eslint-plugin-react/blob/8e1a94b67d081fdc132e9a7e175db3fbf2e02956/tests/lib/rules/jsx-wrap-multilines.js#L1377-L1386